src/core/agent/services/multiSigService.ts

```diff
@@ -1051,7 +1035,7 @@ class MultiSigService extends AgentService {
     } else {
       const time = new Date().toISOString().replace("Z", "000+00:00");
       const [admit, sigs, end] = await this.props.signifyClient.ipex().admit({
-        senderName: multisigSignifyName,
+        senderName: gHab.name,
```

Comment:
I think these can just use `multisigId` too, here and in `submitAdmit`. Can you test that and check for all the IPEX commands?

The reason I want to move these all to identifier is to repurpose signifyName as the displayName and not have inconsistencies if names fail to update etc